apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx

```diff
@@ -28,7 +29,7 @@ import {
   getAccountName,
 } from "@ledgerhq/live-common/lib/account/index";
```

Comment:
i know it's not you, but this import with a lib is bad, it should be `@ledgerhq/live-common/lib/account/index`

Reply:
@haammar-ledger i think it's coming from your hotfix ?

Reply:
Hmm I don't see that, but in any case all wrong paths that target `lib` can all be fixed without second thoughts :+1: